.clang-format

```diff
@@ -1,30 +1,150 @@
 ---
-BasedOnStyle: Google
-IndentWidth: 4
-ColumnLimit: 120
-UseTab: Never
-#BreakBeforeBraces: Allman
----
-Language: Cpp
-PointerAlignment: Right
-DerivePointerAlignment: false
-# Function arguments should either be on one line, or should be each on their own line
-BinPackParameters: false
-BinPackArguments: false
-# Indent nested preprocessor directives
-IndentPPDirectives: AfterHash
-# Short conditionals are allowed:
-# case xyz: blah; break;
-# if (foo) return;
-# while (blah) {}
+Language:        Cpp
```

Comment:
Should this be "C"?

Reply:
It was `Cpp` in the previous version of the file.  Clang-Format [docs](https://releases.llvm.org/8.0.0/tools/clang/docs/ClangFormatStyleOptions.html) don't show `C` as a valid option.